.travis/config.sh

```diff
@@ -0,0 +1,13 @@
+export LSR_MOLECULE_DEPS='-rmolecule_requirements.txt'
+
+case "x${TRAVIS_PYTHON_VERSION}" in
+  x3.5)
```

Comment:
Since this code seems to run in the CI environment, can we just check here that `$TRAVIS_PYTHON_VERSION` is the system python version? Then 3.5 does not need to be hardcoded here.

Reply:
Sorry, I was wrong previously and misunderstood the previous comment. `TRAVIS_PYTHON_VERSION` is generally different from system python version. Instead of hardcoded 3.5 it will be better to use something like:
```bash
if [[ "${TRAVIS_PYTHON_VERSION}" == $(/usr/bin/python3 -c 'import sys; print("{}.{}".format(sys.version_info.major, sys.version_info.minor))') ]]; then
  LSR_EXTRA_PACKAGES='python3-selinux'
fi
```

Reply:
I suggest to make the `/usr/bin/python3 -c 'import sys; print("{}.{}".format(sys.version_info.major, sys.version_info.minor))'` call a shell function to avoid such a long one-liner in the if condition.

Reply:
After some consideration it was decided to install `python3-selinux` on all distros and silently ignore "package missing" errors. This handles the situation when system python bumps to newer version, i.e. 3.6, which cause that `python3-selinux` will not be installed in a 3.5 job, which is a job in which we run molecule.

But there is an another problem: in case of system python bump, even if `python3-selinux` will be installed in 3.5 job, selinux C binding will not be found, which results in infamous `Aborting, target uses selinux but python bindings (libselinux-python) aren't installed!` error message.

The (imho) cleanest solution to this problem is [Allow to filter tox environment for system python](https://github.com/tox-dev/tox-travis/issues/128) proposed by @tyll.

Reply:
I believe that we will see a system python bump only when Travis switches to a new major release of Ubuntu, which will not happen that often.